src/search_engines/hyperscan.cc

```diff
@@ -334,6 +330,12 @@ void hyperscan_cleanup(SnortConfig* sc)
 static Mpse* hs_ctor(
     SnortConfig* sc, class Module*, const MpseAgent* a)
 {
+    if ( !scratch_registered )
+    {
+        scratch_index = SnortConfig::request_scratch(scratch_setup,
+                            scratch_cleanup);
```

Comment:
Nit - 4 space indent

Actually, this can go on one line.  We have limit of 99 chars before newline.
May apply to the other places too.